### ISSUE TITLE
Is it possible to hide the white space at the top of the page?

### ISSUE THREAD

**Issue by i-iooi-i** [Akifyss/obsidian-border#330]
![image](https://github.com/user-attachments/assets/fdb37321-8991-44f4-8243-c91565c81d8f)

This area looks tall, and it's not sure what is hidden here. 
Is there a way to shorten the height of this area? Or customize the height.

**Comment by Akifyss** [comment]
I estimate this is not an issue with the theme; I suspect it may be caused by a certain plugin. Can you try it in the sandbox?

**Comment by i-iooi-i** [comment]
> I estimate this is not an issue with the theme; I suspect it may be caused by a certain plugin. Can you try it in the sandbox?

Thanks for the reminder, I think I found the reason.

`![image](https://github.com/user-attachments/assets/5af470f0-b8f2-4209-a86f-a7a8ccb59237)`
![image](https://github.com/user-attachments/assets/6803457a-44ae-4ca0-86f8-8621e34a4f4a)

I created a new vault, added properties at the top of the page, and then in Settings Editor I hid the properties.
The top area will get higher.

![image](https://github.com/user-attachments/assets/2a1e841c-042b-4456-b3eb-75fff02c11cb)

When I remove all the attributes, the area becomes the right height. 😂😂
 Thank you very much, this should be an issue brought by OB.
